aerogear/graphql-metadata
# [WIP] Migrate package to use TypeScript

This PR was created to move from JavaScript to TypeScript.

## Verification

The best place to verify this is using Graphback:

1. Run `yarn clean && yarn && yarn build` to refresh package dependencies and clear old builds files.
1. From this project, run `npm link`
2. In Graphback root, run `npm link graphql-metadata`
3. Run a Graphback template. It should run successfully.
